### PR TITLE
Switched to remote nodejs-example template & secure storage tweak.

### PR DESCRIPTION
**Pull request by rhopp**
# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by rhopp** [comment]
Do not merge yet... Doing some debugging of pr check job.

**Comment by rhopp** [comment]
testPR

**Comment by rhopp** [comment]
testPR

**Comment by rhopp** [comment]
testPR

**Comment by rhopp** [comment]
testPR

**Comment by rhopp** [comment]
testPR

**Comment by rhopp** [comment]
testPR

**Comment by rhopp** [comment]
testPR

**Comment by rhopp** [comment]
testPR

**Comment by rhopp** [comment]
PR check job seems stable enough (I had some failures, because CDK was unable to start from some weird reason). This PR is ready for review. @jrichter1, anyone?

File: test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/utils/SecureStorage.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
 import org.jboss.reddeer.common.wait.WaitUntil;
 import org.jboss.reddeer.common.wait.WaitWhile;
 import org.jboss.reddeer.core.condition.JobIsRunning;
+import org.jboss.reddeer.core.exception.CoreLayerException;
```
Comment (jrichter1) [inline test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/utils/SecureStorage.java]: Doesn't look like it's being used.

File: test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/utils/SecureStorage.java [hunk 2]
```diff
@@ -135,36 +146,42 @@ private static void waitForLoadingConnectionDetails(AbstractOpenShiftConnection
...
-		
+		boolean exists = secureStoragePreferencePage.passwordExists("[Default Secure Storage]", getPluginId(serverType),
+				server, username);
+
 		workbenchPreferenceDialog.ok();
-		
-		assertTrue(shouldExist? "Password wasn't present in secure storage while it." : "Password was present in secure storage but shouldn't.", 
-				shouldExist == exists);
+
+		assertTrue(shouldExist ? "Password wasn't present in secure storage while it."
```
Comment (jrichter1) [inline test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/utils/SecureStorage.java]: while it should, perhaps?

File: test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/utils/SecureStorage.java (hunk 1 above)
Comment (rhopp) [inline test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/utils/SecureStorage.java]: Done.

File: test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/utils/SecureStorage.java (hunk 2 above)
Comment (rhopp) [inline test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/utils/SecureStorage.java]: Done.

**Comment by jrichter1** [comment]
you get a +1, sir

**Pull request merged**